tasks/ThrusterTask.cpp

```diff
@@ -95,14 +93,13 @@ void ThrusterTask::cleanupHook()
     node->Fini();
 }
 
-void ThrusterTask::setGazeboModel(WorldPtr _world,  ModelPtr _model, sdf::ElementPtr _plugin)
+void ThrusterTask::setGazeboThruster( ModelPtr model )
```

Comment:
Should I change setGazeboThruster() and setGazeboLaserScan() to this ?

    setGazeboModel(WorldPtr _world,  ModelPtr _model, sdf::ElementPtr _plugin)